src/main/java/com/mealmentor/enterprise/MealMentorController.java

```diff
@@ -105,25 +111,24 @@ public String saveMeal(MealItem mealItem){
 
     @GetMapping("/recipeNameAutocomplete")
     @ResponseBody
-    public List<RecipeLabelValue> recipeNameAutocomplete(@RequestParam(value="term", required = false, defaultValue="") String term) {
-
-        List <RecipeLabelValue> allRecipeNames= new ArrayList<RecipeLabelValue>();
-
-    try {
-        List<Recipe> recipes = mealPlanService.fetchRecipes(term);
-        for (Recipe recipe : recipes)
-        {
-            RecipeLabelValue recipeLabelValue = new RecipeLabelValue();
-            recipeLabelValue.setLabel(recipe.getName());
-            recipeLabelValue.setValue(recipe.getId());
-            allRecipeNames.add(recipeLabelValue);
+    public List<RecipeLabelValue> recipeNameAutocomplete(@RequestParam(value = "term", required = false, defaultValue = "") String term) {
+
+        List<RecipeLabelValue> allRecipeNames = new ArrayList<>();
+
+        try {
+            List<Recipe> recipes = mealPlanService.fetchRecipes(term);
+            for (Recipe recipe : recipes) {
+                RecipeLabelValue recipeLabelValue = new RecipeLabelValue();
+                recipeLabelValue.setLabel(recipe.getName());
+                recipeLabelValue.setValue(recipe.getId());
+                allRecipeNames.add(recipeLabelValue);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+            return new ArrayList<>();
```

Comment:
I removed the `RecipeLabelValue` as a parameter from the ArrayList method declaration since you've already mentioned it once in the List<> declaration. It serves no purpose and increases technical debt so I removed it. It is okay both ways but it's a good practice to keep it minimal.